tests/core/test_rtdb.py

```diff
@@ -1,21 +1,351 @@
 """Tests for the runtime database.
 
```

Comment:
Will the test influence the real redis database?

Reply:
The `redis_proc` fixture of pytest_redis will start a Redis instance for test, so it will not influence the real redis database. And if the host and port for `redis_proc` are set to the same with real redis database, there will be a conflict causing the test to fail.